tests/load/pipeline/test_filesystem_pipeline.py

```diff
@@ -299,30 +301,36 @@ def count(*args, **kwargs) -> Any:
         "hiphip": counter("Hurraaaa"),
     }
     now = pendulum.now()
-    os.environ["DESTINATION__FILESYSTEM__BUCKET_URL"] = "file://_storage"
+    kwargs = {}
+    if default_buckets_env.startswith("file://"):
```

Comment:
maybe I'm missing something but we decided to make path just before `put_file` in copy job and only for local filesystem.

Reply:
@rudolfix sorry I've been busy with other two prs, I will do it soon :)

Reply:
@rudolfix I created a separate PR for this https://github.com/dlt-hub/dlt/pull/1267

Reply:
Merged.